src/widget/stock_summary.rs

```diff
@@ -368,29 +377,30 @@ impl CachableWidget<StockState> for StockSummaryWidget {
 
                         Block::default()
                             .borders(Borders::LEFT)
-                            .border_style(Style::default().fg(Color::Blue))
+                            .border_style(Style::default().fg(THEME.border_axis))
                             .render(volume_chunks, buf);
 
                         volume_chunks.x += 1;
 
                         BarChart::default()
                             .bar_gap(0)
                             .bar_set(bar::NINE_LEVELS)
-                            .style(Style::default().fg(Color::DarkGray))
+                            .style(Style::default().fg(THEME.foreground_inactive))
                             .data(&volumes)
                             .render(volume_chunks, buf);
                     }
                 }
             }
 
             Chart::new(datasets)
+                .style(Style::default().bg(THEME.background()))
                 .block(Block::default().border_style(Style::default()))
                 .x_axis(Axis::default().bounds(state.x_bounds(start, end, &data)))
                 .y_axis(
                     Axis::default()
                         .bounds(state.y_bounds(min, max))
                         .labels(state.y_labels(min, max))
-                        .style(Style::default().fg(Color::LightBlue)),
```

Comment:
And this is no longer `LightBlue`

Reply:
Actually, this one seems okay to me, it actually addresses an issue I wanted to bring up a while ago.
| ![Screenshot_20210217_023000](https://user-images.githubusercontent.com/16881812/108143463-1d2a1780-70c8-11eb-8c38-cbfc03d7bdf6.png) | ![Screenshot_20210217_023040](https://user-images.githubusercontent.com/16881812/108143486-287d4300-70c8-11eb-848f-9c99fb852997.png) |
| - | - |
| `0.11.0` | [`#69`](https://github.com/tarkah/tickrs/pull/69) |